en-US/Characters_Codex/06_Spellcasting/Spell_Descriptions/B/Burning_Hands.md

```diff
@@ -9,7 +28,7 @@ _1st-level evocation_
 **Duration:** Instantaneous
 
 Flames shoot from your hands.
-Each creature in the area must make a Dexterity saving throw and takes `3d6` fire damage on a failed save, or half as much damage on a successful one.
+Each creature in a 15-foot cone must make a Dexterity saving throw and takes `3d6` fire damage on a failed save, or half as much damage on a successful one.
```

Comment:
"area" seems to be another term we have to scan for. 🥲

Reply:
Yeah, I would suggest removing it unless that makes the spell difficult to understand.